lib/xebow/animation/static.ex

```diff
@@ -0,0 +1,114 @@
+defmodule Xebow.Animation.Static do
```

Comment:
This is a pretty bad name for what it represents... (a static animation? lol). Was thinking maybe "predefined" fits better? I dunno in other places (mostly docs) I have been calling the other animation types "dynamic" animations so I went with "Static" for these guys.

Reply:
:thinking: I think `Static` works. Maybe @doughsay has an opinion on this one.